packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -2759,9 +2760,19 @@ function pushStyle(
         rules: ([]: Array<Chunk | PrecomputedChunk>),
         hrefs: [stringToChunk(escapeTextForBrowser(href))],
         sheets: (new Map(): Map<string, StylesheetResource>),
+        nonce,
       };
       renderState.styles.set(precedence, styleQueue);
     } else {
+      if (__DEV__) {
+        if (nonce !== styleQueue.nonce) {
```

Comment:
`nonce` is effectively global - should I keep it on the `renderState` instead of on the per-precedence `styleQueue` then?